edgelet/iotedge/src/main.rs

```diff
@@ -115,6 +147,89 @@ fn run() -> Result<(), Error> {
     let mut tokio_runtime = tokio::runtime::Runtime::new().context(ErrorKind::InitializeTokio)?;
 
     match matches.subcommand() {
+        ("check", Some(args)) => {
+            let proxy = std::env::var("HTTPS_PROXY")
```

Comment:
It seems like there is a lot going on in this match leg. Is it possible to move some of this to the `Check` command?

Reply:
Done.